File: PRAK104-2610817120007-AuliaRahmah.c
#include <stdio.h>

int main() {
    int shoes_a = 400000;
    int shoes_b = 350000;
    int discount_a = 13;
    int discount_b = 21;

    int price_a_discount = shoes_a - (shoes_a * discount_a / 100);
    int price_b_discount = shoes_b - (shoes_b * discount_b / 100);

    printf("Harga sepatu A adalah %d\n", shoes_a);
    printf("Harga sepatu B adalah %d\n", shoes_b);
    printf("Sepatu A mendapat diskon %d%% sehingga harganya menjadi %d\n", discount_a, price_a_discount);
    printf("Sepatu B mendapat diskon %d%% sehingga harganya menjadi %d\n", discount_b, price_b_discount);
    
    return 0;
}

File: PRAK104-2610817120007-AuliaRahmah.py
shoes_a = 400000
shoes_b = 350000

discount_a = 13
discount_b = 21

price_a_discount = int(shoes_a - (shoes_a * discount_a / 100))
price_b_discount = int(shoes_b - (shoes_b * discount_b / 100))

print(f"Harga sepatu A adalah {shoes_a}")
print(f"Harga sepatu B adalah {shoes_b}")
print(f"Sepatu A mendapat diskon {discount_a}% sehingga harganya menjadi {price_a_discount}")
print(f"Sepatu B mendapat diskon {discount_b}% sehingga harganya menjadi {price_b_discount}")